src/docs/concepts/index.adoc

```diff
@@ -13,6 +13,14 @@ Micrometer is _not_ a distributed tracing system or an event logger. In Micromet
 For better understanding the differences among these different types of systems (Metrics, Distributed Tracing, and Logging) we recommend Adrian Cole's talk, titled https://www.dotconferences.com/2017/04/adrian-cole-observability-3-ways-logging-metrics-tracing[Observability
 3 Ways].
 
+== Dependencies
+
+The `micrometer-core` module aims to have minimal dependencies. It does not require any third-party dependencies on the classpath at compile time for applications using Micrometer.
```

Comment:
I don't know how clear this is. I by third-party dependencies I mean non-Micrometer dependencies. Suggestions for better wording welcome. Would the following be more clear?
> Only Micrometer dependencies are required on the compile-time classpath to use Micrometer in your application.

Reply:
How about this?
>It does not require any third-party (non-Micrometer) dependencies